lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -29,8 +29,10 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
     return {
       category: 'Performance',
       name: 'external-anchors-use-rel-noopener',
-      description: 'Site opens external anchors using rel="noopener".',
-      helpText: 'Anchors that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener" target="_blank">opening page\'s performance does not suffer.</a>',
+      description: 'Site opens external anchors using rel="noopener"',
+      helpText: 'Open new tabs using <code>rel="noopener"</code> to improve performance and ' +
```

Comment:
@paulirish is there a way to keep the multiline from donking up my URLs?

When I format like this:

<img width="763" alt="screen shot 2016-12-16 at 1 37 26 pm" src="https://cloud.githubusercontent.com/assets/4713486/21279387/d96643a0-c394-11e6-952d-95d76eaa0bf2.png">


It adds space to my URL (see the hover at bottom of Chrome window):

![screen shot 2016-12-16 at 1 35 56 pm](https://cloud.githubusercontent.com/assets/4713486/21279366/b3665ff0-c394-11e6-913d-6b10985161bb.png)